tests/test_parse.py

```diff
@@ -7,16 +8,17 @@
     "Recruiter",
     "DevOps",
     "Digital",
-    "Data"
+    "Data",
 ]  # frontend is web development sometimes
 
 
-def test_get_all_courses():
-    all_courses = get_all_courses()
+@pytest.mark.asyncio
+async def test_get_all_courses():
+    all_courses = await get_all_courses()
 
     course_names = [course.name for course in all_courses]
 
     for course in FOR_SURE_THIS_COURSES:
         assert any(
             course.lower() in course_name.lower() for course_name in course_names
-        ), f"Course '{course}' have not been parsed"
+        ), f"Course '{course}' has not been parsed"
```

Comment:
The assertion checks if any of the expected course names are present in the parsed course list. Consider adding more descriptive error messages or handling exceptions to improve test robustness.